tests/test_authz_authorizer.py

```diff
@@ -34,7 +35,7 @@ def test_validate_one_with_expired(self, full_access_authz_payload: dict) -> Non
         expired_timestamp = int((datetime.now(timezone.utc) - timedelta(seconds=1)).timestamp())
         with pytest.raises(Unauthorized):
             Authorizer(
-                Authorizer.sign_authz(
+                jwt_utils.encode_jwt(
```

Comment:
Sometimes you use the module, sometimes the function directly - I would try to be consistent 😉